### ISSUE TITLE
Tremolo staccato: Default positioning is incorrect when stem points upwards

### ISSUE THREAD

**Issue by Mioriarty** [musescore/MuseScore#20106]
### Issue type

Engraving bug (incorrect score rendering)

### Bug description

When you write a basic note with its stem pointing upwards and you add tremmolo staccato (so multiple staccato dots below the note head) they get display too close to the node head. When the stem points downwards, this does not happen as seen in the screenshot.
I am on a M2 Macbook Air running Sonoma 14.0.

### Steps to reproduce

1. Add a note to any staff where the stem points upwards
2. Navigate in paletts to tremelo and add one of teh staccato markings in this category.

### Screenshots/Screen recordings

Next to every note with tremolo staccato I added a note with regular staccato showing where it should be positioned.
<img width="1111" alt="" src="https://github.com/musescore/MuseScore/assets/22431461/5e0a71d7-90a3-4b93-a051-83f5f0378cb5">

### MuseScore Version

OS: macOS 14.0, Arch.: x86_64, MuseScore version (64-bit): 4.1.1-232071203, revision: github-musescore-musescore-e4d1ddf

### Regression

I don't know

### Operating system

macOS 14.0

**Comment by oktophonie** [comment]
These are not staccatos: they indicate tremolo subdivisions. They are only supposed to appear above notes, where they are generally positioned correctly. See the discussion here: https://github.com/musescore/MuseScore/pull/10000

Not to say that the positioning couldn't be improved in cases where a user insists they go underneath noteheads, at some point.

**Comment by Mioriarty** [comment]
Oh I see. But I defeinitely saw it already once as a staccato marking on tremolo (I am pretty sure) So it might actually be great, if the styling could be fixed :)

**Comment by oktophonie** [comment]
People definitely indicate staccatos that way. But that's not what this symbol is for. (The size of dots doesn't even match normal staccatos)

**Comment by Mioriarty** [comment]
Yeah, but the proper way if you wanted Staccato tremolo would be just writing an ordinary staccato mark below tremolo note, right?

**Comment by SofNotenwart** [comment]
I've never seen so called tremolo dots (amateur violinist since 60 years).  What you can have, however, is tremolo with repeated articulation.  The latter can be staccato, detache etc.  see excerpt from "Behind Bars" by E. Gould.
![RepeatedArticulation](https://github.com/musescore/MuseScore/assets/33205441/74f095d2-8fdd-4f33-aa52-7fdc00a7798d)
